packages/serverless-deploy-iam/bin/app.ts

```diff
@@ -481,7 +481,17 @@ export class ServiceDeployIAM extends cdk.Stack {
         {
           name: "IAM",
           resources: [(serviceRole.type as Role).roleArn],
-          actions: ["iam:PassRole"],
+          actions: [
+            "iam:PassRole", 
+          ],
+        },
+        {
+          name: "IAM",
+          prefix: `arn:aws:iam::${accountId}:role`,
+          qualifiers: ["*"],
```

Comment:
On a wildcard? :thinking:

Reply:
Good find. Fixed, based on the actual usecase.